fuzz/forwarder/fuzzForwarderCall.go

```diff
@@ -0,0 +1,139 @@
+package fuzzForwarder
+
+import (
+	"fmt"
+	"strings"
+
+	vmi "github.com/ElrondNetwork/elrond-go/core/vmcommon"
+)
+
+func (pfe *fuzzExecutor) executeCallPrintLogs(toIndex int) error {
+	el, _ := pfe.getAllExpectedLogs(toIndex)
+	fmt.Println(el)
+	output, err := pfe.executeCall(toIndex, "")
+	if err != nil {
+		return err
+	}
+	for _, log := range output.Logs {
+		fmt.Println(pfe.arwenTestExecutor.LogToString(log))
+	}
+
+	return nil
+}
+
+func (pfe *fuzzExecutor) executeCallCheckLogs(toIndex int) error {
+	expectedLogs, err := pfe.getAllExpectedLogs(toIndex)
+	if err != nil {
+		return err
+	}
+	expectLogs := fmt.Sprintf(`,
+		"expect": {
+			"out": [],
+			"status": "",
+			"logs": [%s
+			]
+		}
+	`,
+		expectedLogs,
+	)
+	_, err = pfe.executeCall(toIndex, expectLogs)
```

Comment:
I think you should also check if the call did what it was supposed to do. It's good that we can know when sync/async/transfer execute call breaks arwen but it would be nice if we would also detect cases where async call (for example) did not transfer the value, or trasfer execute only transferred the value and exited silently without executing the function